app/layouts/default.vue

```diff
@@ -1,9 +1,11 @@
 <template>
   <div class="app">
     <layout-header-default />
-    <panel class="warning-panel" v-if="config.public.test_env">
-      Test Version. Content May Not Persist.
-    </panel>
+
+    <widgets-notification messageType="warning" messageText="Test Version - Data May Not Persist" :modality="true" v-if="config.public.test_env" > 
```

Comment:
If we aren't using/needing the modal here, remove the content and set modality to false please.